packages/patternfly-4/react-core/src/components/Card/Card.md

```diff
@@ -362,3 +362,17 @@ HoverableCard = () => (
   </Card>
 );
 ```
+
+## Card compact example
+```js
+import React from 'react';
+import { Card, CardHeader, CardBody, CardFooter } from '@patternfly/react-core';
+
+HoverableCard = () => (
+  <Card isCompact={true}>
```

Comment:
usually if the prop is true we leave off the `={true}` part

Reply:
Ok, cool. I kept it since the previously written test had it, thinking you guys liked it more explicit. I'll take it off of both. :) Thanks!!